src/app/georeferencer/qgsgeorefmainwindow.cpp

```diff
@@ -214,59 +216,126 @@ void QgsGeoreferencerMainWindow::openRaster( const QString &fileName )
   }
 
   QgsSettings s;
+  QString provider;
+  QString uri;
   if ( fileName.isEmpty() )
   {
-    QString dir = s.value( QStringLiteral( "/Plugin-GeoReferencer/rasterdirectory" ) ).toString();
+    QString dir = s.value( QStringLiteral( "/Plugin-GeoReferencer/lastdirectory" ) ).toString();
```

Comment:
can you create a setting in the registry for this?
I guess it makes sense to rename it at the same time.

Reply:
@3nids 
Can you give an example of how we should be doing that now?